pkgs/development/libraries/linenoise/create-pkg-config-file.sh

```diff
@@ -0,0 +1,13 @@
+cat <<EOF > linenoise.pc
+prefix=$out
+exec_prefix=\${prefix}
+libdir=\${exec_prefix}/lib
+includedir=\${prefix}/include
+
+Name: linenoise
+Description: A minimal, zero-config, BSD licensed, readline replacement.
+Requires: 
+Version: git-2016-09-30
```

Comment:
Also, this is out of sync with the expression. Maybe generate this inline or use `substituteAll` to ensure consistency.